app/cli/internal/action/attestation_push.go

```diff
@@ -131,12 +131,14 @@ func (action *AttestationPush) Run(ctx context.Context, attestationID string, ru
 	// Indicate that we are done with the attestation
 	action.c.CraftingState.Attestation.FinishedAt = timestamppb.New(time.Now())
 
-	renderer, err := renderer.NewAttestationRenderer(action.c.CraftingState, action.keyPath, action.cliVersion, action.cliDigest, renderer.WithLogger(action.Logger))
+	renderer, err := renderer.NewAttestationRenderer(action.c.CraftingState, action.keyPath, action.cliVersion, action.cliDigest,
+		renderer.WithLogger(action.Logger),
+		renderer.WithCPSigningClient(pb.NewSigningServiceClient(action.CPConnection)))
```

Comment:
eventually we should probably have a `signer` interface that can be implemented either via pathToKey or remote signing like this one. For now this is fine though :)